khal/controllers.py

```diff
@@ -470,7 +471,10 @@ def edit_event(event, collection, locale, allow_quit=False, width=80):
                 until = prompt('until (or "None")', until)
                 if until == 'None':
                     until = None
-                rrule = parse_datetime.rrulefstr(freq, until, locale)
+                interval = prompt('every (or "None")', interval)
+                if interval == 'None':
```

Comment:
Empty string should also fall back to `None `.

Reply:
It seem to work with an empty string as input like until above.  This is because, I believe, due to the choice condition that changes the default from None to ''.  I can add a test of empty string, of course, do you want me to do that for until, too?

Reply:
I did so in my last comment, also addressed a couple of defects in editing an existing recurring event.